inc/class-upgrade.php

```diff
@@ -829,6 +830,82 @@ private function upgrade_165() {
 		add_action( 'init', [ $this, 'reset_og_settings_to_default_values' ], 100 );
 	}
 
+	/**
+	 * Performs the 16.8 upgrade.
+	 *
+	 * @return void
+	 */
+	private function upgrade_168() {
+		// Clean out shop_order post types from the indexables table and return which ids were cleaned out.
+		$cleaned_indexable_ids = $this->delete_shop_order_indexables_for_168();
```

Comment:
I would delete the indexables after deleting the prominent words and internal links, so if something goes wrong you don't have orphaned links and prominent words with no corresponding indexable. 

Also, you could rename the function to `delete_indexables_with_object_type`, and pass along the object_type and object_sub_type, so we can reuse this method if a similar issue arises in the future.

Reply:
Makes sense, thanks 👍
Will do something similar, considering I've refactored the code a bit since you posted that message :)